cargo build fails, throws error [E0252], when importing as dependency in new rust project

### Describe the bug

Using reth as a dependency in another cargo project fails and the error "the name `bytes` is defined multiple times" in the file crates/primitives/src/lib.rs. 

error comes from:
line 53:  "...use bytes::{self..." 

&

line 103: "...bloom, bytes, ..."

included the build log error below.

tried building on m2 mac, Debian x86_64 on gcp, and ubuntu running through multipass on m2 mac. All three fail with the same error.

### Steps to reproduce

1. cd example_build
2. cargo init
3. cargo add reth --git https://github.com/paradigmxyz/reth.git
4. cargo build

### Node logs

```text
Compiling reth-primitives v0.1.0-alpha.13 (https://github.com/paradigmxyz/reth#365acf08)
   Compiling tinystr v0.7.2
   Compiling ring v0.17.7
   Compiling litemap v0.7.2
   Compiling base64 v0.13.1
error[E0252]: the name `bytes` is defined multiple times
   --> /home/los/.cargo/git/checkouts/reth-36d3ea1d1152b20c/365acf0/crates/primitives/src/lib.rs:103:33
    |
53  | pub use bytes::{self, Buf, BufMut, BytesMut};
    |                 ---- previous import of the module `bytes` here
...
103 |     self, address, b256, bloom, bytes, eip191_hash_message, hex, hex_literal, keccak256, ruint,
    |                                 ^^^^^ `bytes` reimported here
    |
    = note: `bytes` must be defined only once in the type namespace of this module
help: you can use `as` to change the binding name of the import
    |
103 |     self, address, b256, bloom, bytes as other_bytes, eip191_hash_message, hex, hex_literal, keccak256, ruint,
    |                                 ~~~~~~~~~~~~~~~~~~~~
```


### Platform(s)

Linux (x86), Mac (Apple Silicon)

### What version/commit are you on?

v0.1.0-alpha.13

### What database version are you on?

n/a

### What type of node are you running?

None

### What prune config do you use, if any?

n/a

### If you've built Reth from source, provide the full command you used

_No response_

### Code of Conduct

- [X] I agree to follow the Code of Conduct

--- a/crates/primitives/src/lib.rs
+++ b/crates/primitives/src/lib.rs
@@ -50,7 +50,6 @@ pub use block::{
     Block, BlockBody, BlockHashOrNumber, BlockId, BlockNumHash, BlockNumberOrTag, BlockWithSenders,
     ForkBlock, RpcBlockHash, SealedBlock, SealedBlockWithSenders,
 };
-pub use bytes::{self, Buf, BufMut, BytesMut};
 pub use chain::{
     AllGenesisFormats, BaseFeeParams, BaseFeeParamsKind, Chain, ChainInfo, ChainSpec,
     ChainSpecBuilder, DisplayHardforks, ForkBaseFeeParams, ForkCondition, ForkTimestamps,
@@ -100,9 +99,11 @@ pub use withdrawal::Withdrawal;
 // Re-exports
 pub use self::ruint::UintTryTo;
 pub use alloy_primitives::{
-    self, address, b256, bloom, bytes, eip191_hash_message, hex, hex_literal, keccak256, ruint,
-    Address, BlockHash, BlockNumber, Bloom, BloomInput, Bytes, ChainId, Selector, StorageKey,
-    StorageValue, TxHash, TxIndex, TxNumber, B128, B256, B512, B64, U128, U256, U64, U8,
+    self, address, b256, bloom, bytes,
+    bytes::{Buf, BufMut, BytesMut},
+    eip191_hash_message, hex, hex_literal, keccak256, ruint, Address, BlockHash, BlockNumber,
+    Bloom, BloomInput, Bytes, ChainId, Selector, StorageKey, StorageValue, TxHash, TxIndex,
+    TxNumber, B128, B256, B512, B64, U128, U256, U64, U8,
 };
 pub use reth_ethereum_forks::*;
 pub use revm_primitives::{self, JumpMap};

